set.mm

```diff
@@ -29221,6 +29221,16 @@ choice between (what we call) a "definitional form" where the shorter
   vex $p |- x e. _V $=
     ( cv cvv wcel weq equid df-v abeq2i mpbir ) ABCDAAEZAFJACAGHI $.
 
+  ${
+    elv.1 $e |- ( x e. _V -> ph ) $.
+    $( New way to shorten some proofs.  Inference forms (with ` |- A e. _V `
+       hypotheses) of the general theorems (proving ` |- ( A e. V -> `
+       assertions) may be superfluous.  (Contributed by Peter Mazsa,
+       13-Oct-2018.) $)
+    elv $p |- ph $=
+      ( cv cvv wcel vex ax-mp ) BDEFABGCH $.
+  $}
```

Comment:
Not sure it shortens sufficiently many proofs to be moved to Main... But I guess you haven't tried the scripts/min.cmd yet?
If it is to stay, then the comment should be adapted. Maybe:
> Technical lemma used to shorten proofs.  If a proposition is implied by ` x e. _V ` (which is true, see ~ vex ), then it is true.

Reply:
I primarily moved it because I use it in my mathbox. But as I wrote in my comment, 301 proofs can be shortened using ~elv! I expect a saving of about 3000 bytes. But I will change the comment as you propose.

Reply:
Comment changed, minimization, as announced, in a next PR.